pybossa/view/projects.py

```diff
@@ -2951,6 +2951,34 @@ def form_to_db(config):
             in six.iteritems(DbFormConverter.form_to_db_field_map)
         }
 
+def process_quiz_mode_request(project):
+    db_current_quiz_config = project.get_quiz()
+
+    if request.method == 'GET':
+        return ProjectQuizForm(**DbFormConverter.db_to_form(db_current_quiz_config))
```

Comment:
This is part of the previous request and I think you and Deep discussed about it, but `DbFormConverter` sounds a lot more generic than it is. Also not sure why we need it and can't have the same names for the db fields and the form fields.

Reply:
One of the DB field names is `pass` and that's a python reserved word so I can't use it for a form field name.

Reply:
does it have to be called `pass`?

Reply:
I changed it to `passing`.